Data-Science/MLflow/bike-sharing-prediction.ipynb

```diff
@@ -38,7 +27,34 @@
     "        print(\"Deployment successful!\")\n",
     "    except subprocess.CalledProcessError as e:\n",
     "        print(f\"Deployment failed. Error: {e}\")\n",
-    "        \n",
+    "\n",
+    "\n",
+    "## update kserve-objectstore-s3 secret\n",
+    "with open('kserve/kserve-objectstore-s3-secret.yaml', 'r') as file:\n",
+    "    s3_yaml_data = yaml.safe_load(file)\n",
+    "\n",
+    "# Convert YAML to JSON\n",
+    "s3_json_data = json.dumps(s3_yaml_data)\n",
+    "s3_data = json.loads(s3_json_data)\n",
+    "\n",
+    "s3_data[\"stringData\"][\"AWS_ACCESS_KEY_ID\"] = os.environ['AUTH_TOKEN']\n",
+    "s3_data[\"stringData\"][\"AWS_SECRET_ACCESS_KEY\"] = \"s3\"\n",
+    "s3_data[\"stringData\"][\"MLFLOW_S3_ENDPOINT_URL\"] = os.environ['MLFLOW_S3_ENDPOINT_URL']\n",
+    "s3_data[\"stringData\"][\"MLFLOW_S3_IGNORE_TLS\"] = \"true\"\n",
+    "s3_data[\"stringData\"][\"namespace\"] = os.environ['USER']\n",
```

Comment:
Those 3 fields are nor needed for KServe and can be removed, since this secret is used only for KServe.

Reply:
But they also should be removed from Data-Science/MLflow/kserve/kserve-objectstore-s3-secret.yaml

Reply:
right, my bad, done!